ansible/templates/dehydrated.j2

```diff
@@ -0,0 +1,17 @@
+##########################################################################
+# See the dehydrated docs for all config options:
+# https://github.com/lukas2511/dehydrated/blob/master/docs/examples/config
+##########################################################################
+
+# Path to certificate authority (default: https://acme-v01.api.letsencrypt.org/directory)
+#CA="https://acme-v01.api.letsencrypt.org/directory"
+
+# Path to certificate authority license terms redirect (default: https://acme-v01.api.letsencrypt.org/terms)
+#CA_TERMS="https://acme-v01.api.letsencrypt.org/terms"
+
+# Staging servers, for testing
+CA="https://acme-staging.api.letsencrypt.org/directory"
```

Comment:
can we update these to be the live servers? also, how do you feel about omitting all uncommented config lines? (maybe leave the link to docs tho)